doc_source/ContainerInsights-Prometheus-Sample-Workloads-appmesh.md

```diff
@@ -10,171 +10,56 @@ You must add the **AWSAppMeshFullAccess** policy to the IAM role for your Amazon
-
-1. Create App Mesh testing traffic by entering the following command:
-
-   ```
-   curl https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/prometheus-beta/k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/sample_traffic/appmesh/appmesh_traffic_sample.yaml | 
-   sed "s/{{appmeshname}}/$APPMESH_NAME/g" | 
-   sed "s/{{namespace}}/$APPMESH_SAMPLE_TRAFFIC_NAMESPACE/g" | 
-   kubectl apply -f -
-   ```
-
-1. \(Optional\) Install a second App Mesh workload service to generate 404 and 503 response codes by entering the following command:
-
-   ```
-   APPMESH_SAMPLE_ERROR_TRAFFIC_NAMESPACE=appmesh-sample-errortraffic-test
-   ```
-
-   ```
-   curl https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/prometheus-beta/k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/sample_traffic/appmesh/appmesh_traffic_sample_4xx5xx.yaml | 
-   sed "s/{{appmeshname}}/$APPMESH_NAME/g" | 
-   sed "s/{{namespace}}/$APPMESH_SAMPLE_ERROR_TRAFFIC_NAMESPACE/g" | 
-   kubectl apply -f -
-   ```
-
-1. Find the name of the traffic generator by entering the following command:
-
-   ```
-   kubectl get pod -n $APPMESH_SAMPLE_TRAFFIC_NAMESPACE 
-   ```
-
-   The traffic generator is in the NAME column and starts with `traffic-generator`
-
-1. Find the name of the virtual service by entering the following command:
-
-   ```
-   kubectl get virtualservice.appmesh.k8s.aws -n $APPMESH_SAMPLE_TRAFFIC_NAMESPACE
-   ```
-
-1. Remote exec into the traffic generator pod by entering the following comman\. Substitute *traffic\-generator\-name* with the name of the traffic generator that you found in step 9\.
-
-   ```
-   kubectl exec -ti traffic-generator-name bash -n $APPMESH_SAMPLE_TRAFFIC_NAMESPACE
-   ```
-
-1. Run the curl command multiple times, using the name of the virtual service that you found in step 10\.
-
-   ```
-   curl virtual_servie_name:9080;echo
-   ```
-
-1. Exit the pod by entering the following command:
-
-   ```
-   Exit
-   ```
+## Check CloudWatch Metrics and Logs<a name="ContainerInsights-Prometheus-Sample-Workloads-appmesh-cw-check"></a>
 
 1. Open the CloudWatch console at [https://console\.aws\.amazon\.com/cloudwatch/](https://console.aws.amazon.com/cloudwatch/)\.
 
-1. In the AWS Region where your cluster is running, choose **Metrics**\. The App Mesh meetrics are in the **ContainerInsights/Prometheus** namespace\.
+1. In the AWS Region where your cluster is running, choose **Metrics**\. The App Mesh metrics are in the **ContainerInsights/Prometheus** namespace\.
 
 1. To see the CloudWatch Logs events, choose **Log Groups** in the navigation pane\. The events are in the log group **/aws/containerinsights/*your\_cluster\_name*/prometheus**, in the log stream **kubernetes\-pod\-appmesh\-envoyappmesh**\.
```

Comment:
The log stream name should be:
`kubernetes-pod-appmesh-envoy`

Please remove `appmesh` at the end.